python/utils/color.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2016 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from sgtk.platform.qt import QtGui
+
+
+def color_mix(c1, c1_strength, c2, c2_strength):
```

Comment:
I would love for this to be a first class citizen. I actually pulled it from the `tk-multi-pythonconsole` and would use this from there if I could. It's great for writing apps that need to display custom colors that fit within the various DCC's color palettes. Let me know what you think!